src/__tests__/standardMethods.spec.ts

```diff
@@ -27,6 +30,26 @@ function numberGeneratorAPIEndpoint(config: AxiosRequestConfig) {
   }
 }
 
+function countChunks(axiosMock: MockAdapter, url: string) {
+  const counterObject = { value: 0 };
```

Comment:
why do you wrap it in object? can't you just have a `number` variable?

Reply:
Not if you want to return a dynamic number-variable from the function. Need a reference and not a plain value